src/classes/HealthCheckVMapper_TEST.cls

```diff
@@ -119,6 +119,25 @@ public class HealthCheckVMapper_TEST {
         }  
     }
 
+    /****************************************************************************************************************************
+    * @description Test to verify that getHealthCheckViewModel throws AuraHandledException
+    ****************************************************************************************************************************/
+    @isTest 
+    private static void getHealthCheckViewModelException(){
+
+        FeatureParameterMapper.instance = new STUB_FeatureParameterMapperError();
+        HealthCheckDefinitionService.instance = new STUB_HealthCheckDefinitionService();
+
+        try {
+            Test.startTest();
+                HealthCheckVModel healthChkViewMod = HealthCheckVMapper.getInstance().getHealthCheckViewModel();
+            Test.stopTest();
+            System.assert(false, 'Should throw an exception');
+        } catch (AuraHandledException ex) {
+            System.assertEquals('System.AuraHandledException', ex.getTypeName(), 'AuraHandledException should be thrown when exception occurs.');
```

Comment:
This is the right play here since the exception in this case is a passthrough with an arbitrary message.